src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.cs

```diff
@@ -462,14 +318,12 @@ private SolutionState AddProject(ProjectId projectId, ProjectState projectState)
                 }
             }
 
-            var newTrackerMap = CreateCompilationTrackerMap(projectId, newDependencyGraph);
```

Comment:
this logic is now in SolutionCompilationState.AddProject.  Which is does after calling into this method to get the new SolutionState.